HW1/P3/.ipynb_checkpoints/Problem 3-checkpoint.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
This is a very minor complaint, but uploading IPython notebooks makes it a bit difficult to look at code by scrolling down the "Files changed" tab, since the notebooks are so long.